README.md

```diff
@@ -0,0 +1,17 @@
+## OlivApp
+Software para asesoramiento de venta de aceite.
+
+## Definición del problema
+Este proyecto aborda el problema que tienen muchas empresas comercializadoras de aceite a la hora de pronosticar el momento más idóneo para la venta de su producto con el fin de mejorar su rentabilidad.
+
+## Cuales serían los usuarios/empresas que utilizarían este software
+Empresas productoras y comercializadoras de aceite de oliva (cooperativas oleícolas, almazaras privadas, productores particulares, compradores, agentes comerciales...).
+
+## Planteamiento de la solución
+El objetivo de la plataforma es que los usuarios tengan una idea lo más aproximada posible sobre que momento del año puede estar en su nivel más alto el precio del aceite de oliva en origen.
+
+El programa descargará una serie de archivos de la nube para llevar a cabo su predicción. Estos archivos son históricos basados en tablas que recogen los datos de un parámetro deteminado(existencias iniciales, consumo, producción y precipitaciones) en cada mes de un año. Entonces el programa descargará un histórico del año actual de las existencias iniciales, del consumo, de la producción y de las precipitaciones (este parámatro aunque parezca que no pero será importante tenrlo en cuenta ya que influye en la producción de la cosecha) y descargará los mismos históricos pero de los 5/10 años anteriores. A partir de estos documentos el programa comparará los datos (existencias iniciales, consumo, producción y precipitaciones) del año actual con los de años anteriores y a partir de una serie de cálculos obtendrá cual sería el año más parecido al actual y a partir de ahí se basará para realizar su predicción; Esta predicción nos proporcionará en que época/meses del año el coste del aceite de oliva alcanza un mayor precio y por lo tanto sale más rentable su venta.
```

Comment:
De veras, debéis intentar centraros en el problema y ser conscienes de que lo vais a programar. En el momento que dices "una serie de archivos" sin especificar de qué estás hablando, ni dónde están esos archivos, ni por qué lo que contienen esos archivos va a solucionar ningún problema, ni nada más estás dejando bastante claro que estás intentando salir del paso de este objetivo y luego ya se verá.
Por favor, consulta los conceptos y herramientas impartidos en clase para entender el problema de algún colectivo.
En el caso concreto, ni siquiera estás entendiendo que el precio del aceite depende de la cosecha que haya cada año, no de la época del año.

Reply:
Y esto podías haberlo contestado también poniendo los URLs en vez de decir "la nube". Aquí simplemente quieres decir "la web". Si además sabes las direcciones o las instituciones, dilo.

Reply:
:arrow_up: